libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -466,15 +466,17 @@ - (NSArray*) getDevSupportInfos
             @"SDK Version", SALESFORCE_SDK_VERSION,
             @"App Type", [self getAppTypeAsString],
             @"User Agent", self.userAgentString(@""),
-            @"Browser Login Enabled", [SFUserAccountManager sharedInstance].advancedAuthConfiguration != SFOAuthAdvancedAuthConfigurationNone ? @"true" : @"false",
+            @"Browser Login Enabled", [SFUserAccountManager sharedInstance].advancedAuthConfiguration != SFOAuthAdvancedAuthConfigurationNone ? @"YES" : @"NO",
+            @"IDP Enabled", [self idpEnabled] ? @"YES" : @"NO",
+            @"Identity Provider", [self isIdentityProvider] ? @"YES" : @"NO",
```

Comment:
@bhariharan what is the equivalent call to `[self isIdentityProvider]` on Android ?

Reply:
There is no equivalent on `Android`. I wanted to keep it as simple as possible to enable an app as an IDP on `Android`, so all you have to do is uncomment a few lines of code in `AndroidManifest.xml` of your app and your app becomes IDP-capable (see [here](https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/dev/native/NativeSampleApps/SmartSyncExplorer/AndroidManifest.xml#L73)). Since it's in XML it would be hard to get programmatically, but if you really wanted to, you'd have to try to get a handle on the `Manifest` elements from code and traverse the XML looking for that particular element (look [here](https://developer.android.com/reference/android/Manifest.html)). I don't know if it's worth it though.

Reply:
There is an equivalent for `[self idpEnabled]` though on `SalesforceSDKManager` for the SP side.